src/panels/config/integrations/integration-panels/dynalite/dynalite-config-panel.ts

```diff
@@ -0,0 +1,289 @@
+import "@material/mwc-button";
+import "@polymer/app-layout/app-header/app-header";
+import "@polymer/app-layout/app-toolbar/app-toolbar";
+import "@polymer/paper-input/paper-input";
+import "@polymer/paper-listbox/paper-listbox";
+import "@polymer/paper-item/paper-item";
+import type { PaperInputElement } from "@polymer/paper-input/paper-input";
+import type { PolymerChangedEvent } from "../../../../../polymer-types";
+import {
+  css,
+  CSSResultArray,
+  customElement,
+  html,
+  LitElement,
+  property,
+  internalProperty,
+  TemplateResult,
+} from "lit-element";
+import "../../../../../components/ha-card";
+import "../../../../../components/ha-menu-button";
+import "../../../../../layouts/ha-app-layout";
+import "../../../../../components/ha-settings-row";
+import "../../../../../components/ha-paper-dropdown-menu";
+import "../../../../../components/ha-switch";
+import { haStyle } from "../../../../../resources/styles";
+import { HomeAssistant } from "../../../../../types";
+
+@customElement("dynalite-config-panel")
+class HaPanelConfigDynalite extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public narrow!: boolean;
+
+  @internalProperty() private _name = "";
+
+  @internalProperty() private _host = "";
+
+  @internalProperty() private _port = "";
+
+  @internalProperty() private _fade = "";
+
+  @internalProperty() private _active = "";
+
+  @internalProperty() private _auto_discover = "";
+
+  @internalProperty() private _poll_timer = "";
+
+  private _entryData;
+
+  private _activeOptions = ["on", "init", "off"];
+
+  private _activeIndex = 0;
```

Comment:
this should be a property?

Reply:
don't think so. it is only used for the initial selection value but doesn't get updated itself, unless there is something i dont understand

Reply:
in any case, removed now